### DISCUSSION TITLE
Unable to use Grammarly in VSCode

### DISCUSSION THREAD

**Discussion by Alexis-Dorange** [znck/grammarly#395] [Q&A]
Good evening, 
I tried to install Grammarly and it sends the following error message : 
```
Ready!
[Error - 5:56:38 PM] Server initialization failed.
  Message: Request initialize failed with message: fetch failed
  Code: -32603 
[Error - 5:56:38 PM] Starting client failed
  Message: Request initialize failed with message: fetch failed
  Code: -32603
```
I tried using different versions and the same problem occurs every time. 
I can not login or anything since it tries to strat a Grammarly language server without stopping.
What should I do?

**Comment by LiuX2018** [comment]
same problem.

**Comment by jcs090218** [comment]
What node version are you using? 🤔
